app/models/lunch.rb

```diff
@@ -6,11 +6,13 @@ class << self
     include JSON
 
     # Allergies supported in API
+    # https://chalmerskonferens.se/en/api/
```

Comment:
Are we even using this API now?

Reply:
I don't see where? 🤔

Reply:
It's where you find the links to the APIs for each restaurant.